src/synapse/workload.py

```diff
@@ -31,10 +31,12 @@
 SYNAPSE_CONFIG_PATH = f"{SYNAPSE_CONFIG_DIR}/homeserver.yaml"
 SYNAPSE_CONTAINER_NAME = "synapse"
 SYNAPSE_DATA_DIR = "/data"
+SYNAPSE_GROUP = "synapse"
 SYNAPSE_NGINX_CONTAINER_NAME = "synapse-nginx"
 SYNAPSE_NGINX_PORT = 8080
 SYNAPSE_NGINX_SERVICE_NAME = "synapse-nginx"
 SYNAPSE_SERVICE_NAME = "synapse"
+SYNAPSE_USER = "synapse"
```

Comment:
Will this be only used for backups ? Then it should be called `synapse_backup` (or just `backups` ?) in my opinion

Reply:
It is the same user as used by the Synapse server. I think it can confuse if I name it as `synapse_backup` when it is the same one (the 991 uid).

Reply:
It make sense if it's the same